bin/slack-notify.sh

```diff
@@ -5,9 +5,9 @@ set -eo pipefail
 # BRANCH_NAME, BUILD_NUMBER
 
 # defaults and constants
-CHANNEL="sumodev"
```

Comment:
Shall we use the existing channel? It's already well known by users and it usually doesn't have any noise apart from dev related topics. The new channel has also logs from `yar` which will make it harder to locate the builds.

Reply:
I'd be happy to but in the interim it's going to be noisy while we get everything stood up, and I don't want to confuse a non-production event with a production event.  Could we perhaps switch back post-migration to the current channels?